lib/components/artist/related_artists/related_artist.js

```diff
@@ -52,6 +49,19 @@ const styles = StyleSheet.create({
 });
 
 export default Relay.createContainer(RelatedArtist, {
+  initialVariables: { imageWidth: 150, imageHeight: 100 },
+  prepareVariables: (prevVariables) => {
+    const {windowHeight, windowWidth} = Dimensions.get('window');
+    let newWidth = (windowWidth - 60)/2;
+    console.log(newWidth);
+    console.log('window width: ' + windowWidth);
```

Comment:
I'm going to timebox trying to figure out why `Dimensions` is undefined here because it seems very suspicious, especially given the documentation that uses `window`. Perhaps it's something to do with my code.